src/index.ts

```diff
@@ -199,6 +203,8 @@ const plugin = {
     [noTestPrefixesName]: noTestPrefixes,
     [noTestReturnStatementName]: noTestReturnStatement,
     [noImportNodeTestName]: noImportNodeTest,
+    [noImportingVitestGlobalsName]: noImportingVitestGlobals,
+    [preferImportingVitestGlobalsName]: preferImportingVitestGlobals,
```

Comment:
can we remove these rules from recommended rules. It doesn't make sense to have them there.